pytorch_lightning/loggers/wandb.py

```diff
@@ -413,11 +413,11 @@ def log_image(self, key: str, images: List[Any], **kwargs: str) -> None:
         """
         if not isinstance(images, list):
             raise TypeError(f'Expected a list as "images", found {type(images)}')
+        step = kwargs.pop("step", None)
```

Comment:
or maybe we could add `step` as a parameter (default None) and keep `kwargs` completely separate?

Reply:
also mind add an try in the changelog?

Reply:
I thought it did not introduce any actual change to the expected behaviour, does it make sense to add it anyway ? Like "Make step argument work properly in WandbLogger's log_image" ?

Reply:
it's a bug fix, so yes, under the bugfix section.